github-linguist.gemspec

```diff
@@ -13,7 +13,8 @@ Gem::Specification.new do |s|
     "github_repo" => "ssh://github.com/github/linguist"
   }
 
-  s.files = Dir['lib/**/*'] + Dir['ext/**/*'] + Dir['grammars/*'] + ['LICENSE']
+  s.files = Dir['lib/**/*', 'ext/**/*', 'grammars/*', 'LICENSE'].reject {|f| f.match(%r{lib/linguist/linguist\.(so|bundle)}) }
```

Comment:
Not a blocker, but this strikes me as more idiomatic Ruby:

~~~suggestion
  s.files = Dir['{lib,ext}/**/*', 'grammars/*', 'LICENSE'] - Dir['lib/linguist/linguist.{so,bundle}']
~~~

Reply:
🤦 Why didn't I think of that? Thanks. 🙇

Reply:
Always helps to have another set of eyes. 😉